docs/user_guide/method/writing_custom_methods.rst

```diff
@@ -0,0 +1,528 @@
+central-transition selective method, we specify :math:`D=0`.
+
+.. plot::
+    :context: close-figs
+
+    ct_pulseacquire = Method(
+        channels=["87Rb"],
+        magnetic_flux_density=9.4,  # in T
+        rotor_frequency=10000,  # in Hz
+        spectral_dimensions=[
+            SpectralDimension(
+                count=20000,
+                spectral_width=8e3,
+                reference_offset=-3.5e3,
+                events=[
+                    SpectralEvent(transition_query=[{"ch1": {"P": [-1], "D": [0]}}])
+                ]
+            )
+        ]
+    )
+
+We now replace the old ``pulseacquire`` method in the simulator object with our new
+``ct_pulseacquire`` method and re-simulate the spectrum.
+
+.. We simply add this new method to the simulator object, run the
+.. simulation, apply our processing, and plot the data.
+
+.. skip: next
+
+.. plot::
+    :context: close-figs
+    :caption: A simulated central-transition selective spectrum of :math:`{87}^\text{Rb}`. The large number of sidebands from the previous simulation have been suppressed.
+
+    sim.methods = [ct_pulseacquire]
+    sim.config.number_of_sidebands = 70  # Reset number of sidebands for efficiency
+    sim.run()
+
+    processed_data = processor.apply_operations(dataset=sim.methods[0].simulation.real)
+
+    plt.figure(figsize=(6, 4))
+    ax = plt.subplot(projection="csdm")
+    ax.plot(processed_data.real, color="black", linewidth=1)
+    ax.invert_xaxis()
+    plt.tight_layout()
+    plt.show()
+
+Three-Quantum MAS
+-----------------
+
+Now, let’s construct a method to simulate a 3Q-MAS spectrum.
+
+.. plot::
+    :context: close-figs
+
+    mqmas = Method(
+        channels=["87Rb"],
+        magnetic_flux_density=9.4,
+        rotor_frequency=10000,
+        spectral_dimensions=[
+            SpectralDimension(
+                count=128,
+                spectral_width=6e3,  # in Hz
+                reference_offset=-9e3,  # in Hz
+                label="3Q resonances",
+                events=[
+                    SpectralEvent(transition_query=[{"ch1": {"P": [-3], "D": [0]}}])
+                ]
+            ),
+            SpectralDimension(
+                count=256,
+                spectral_width=6e3,  # in Hz
+                reference_offset=-5e3,  # in Hz
+                label="1Q resonances",
+                events=[
+                    SpectralEvent(transition_query=[{"ch1": {"P":[-1], "D": [0]}}])
+                ]
+            )
+        ],
+    )
+
+Now, instead of just one item in the list of spectral dimensions, we
+have two, because 3Q-MAS is a two-dimensional experiment. In the first
+dimension, we are selecting the triple-quantum coherence by specifying a
+transition query of :math:`p=-3` and :math:`d=0`. In the MAS dimension,
+we are selecting the central transition with a transition query of
+:math:`p=-1` and :math:`d=0`.
+
+Again, we add this method to the simulator object, run the simulation, and
```

Comment:
Again, we replace the previous method object, ``ct_pulseaquire``, with ``mqmas``, run the...